src/ucp/core/ucp_context.c

```diff
@@ -546,6 +546,12 @@ static ucs_config_field_t ucp_context_config_table[] = {
    ucs_offsetof(ucp_context_config_t, extra_op_attr_flags),
    UCS_CONFIG_TYPE_BITMAP(ucp_extra_op_attr_flags_names)},
 
+  {"PRIO_EST_NUM_EPS", "20",
```

Comment:
why is it an "estimation" ? 
Maybe it is an upper bound on how many endpoints would be created on specific transport?

Reply:
what if the specific transport is only available? it seems like we need new formula for lanes selection and probably new UCT capabilities for this.

Reply:
pls adjust variable name - MAX_PRIORITY_EPS and field name to max_priority_eps